en/micro-integrator/docs/references/connectors/redis-connector/redis-connector-example.md

```diff
@@ -0,0 +1,453 @@
+
+1. Add address endpoint using with the send mediator to access SimpleStockQuoteService.
+   
+   <img src="../../../../assets/img/connectors/redis-address-endpoint.png" title="Address endpoint" width="500" alt="Address endpoint"/>  
+
+2. Add Header to get Quote from the SimpleStockQuoteService.
+
+   <img src="../../../../assets/img/connectors/redis-header.png" title="Add Header to get Quote" width="500" alt="Add Header to get Quote"/> 
+
+3. Add payload factory to extract selected stock details. In this sample we try to get IBM stock details from the SimpleStockQuoteService.
+   
+   <img src="../../../../assets/img/connectors/redis-payloadfactory.png" title="Add payloadfactory to extract IBM details" width="500" alt="Add payloadfactory to extract IBM details"/> 
+
+4. Initialize the connector.
+    
+    1. Navigate into the **Palette** pane and select the graphical operations icons listed under **Redis Connector** section. Then drag and drop the `init` operation into the Design pane.
+        
+        <img src="../../../../assets/img/connectors/redis-init-drag-and-shop.png" title="Drag and drop init operation" width="500" alt="Drag and drop init operation"/>   
+    
+    2. Add the property values into the `init` operation as shown below. Replace the `redisHost`, `redisPort`, `redisTimeout` with your values.
+        
+        - **redisHost** : The Redis host name (default localhost).. 
+        - **redisPort** : The port on which the Redis server is running (The default port is 6379).
+        - **redisTimeout** : The server TTL (Time to live) in milliseconds.
+    
+        <img src="../../../../assets/img/connectors/redis-init-parameterspng.png" title="Add values to the init operation" width="800" alt="Add values to the init operation"/>
+
+5. Set up the iPush operation. In this operation we can inserts one or more elements to the head of a list.
+
+    1. Navigate into the **Palette** pane and select the graphical operations icons listed under **Redis Connector** section. Then drag and drop the `iPush` operation into the Design pane.
+           
+        <img src="../../../../assets/img/connectors/redis-ipush-operation.png" title="Drag and drop ipush operation" width="500" alt="Drag and drop ipush operation"/>    
+
+    2. In this operation we are going to inserts one or more elements to the head of a list.The lPush operation inserts one or more elements to the head of a list that is stored in a specified key.                                                                                          
+          
+        - **redisKey** : The name of the key where the list is stored.
+        - **redisStrings** : One or more elements that you want to add to the head of the list.
+        
+        While invoking the API, the above two parameters values extracted from the response of SimpleStockQuoteService. Then insert as a input value.
```

Comment:
Then insert as a input value --> Then they are populated as an input value for the redis `iPush` operation.

Reply:
Updated.